tools/lint-md.js

```diff
@@ -6024,29 +6033,29 @@ var minimist = function (args, opts) {
         }
         else if (/^-[^-]+/.test(arg)) {
             var letters = arg.slice(1,-1).split('');
-
+            
             var broken = false;
             for (var j = 0; j < letters.length; j++) {
                 var next = arg.slice(j+2);
-
+                
                 if (next === '-') {
                     setArg(letters[j], next, arg);
                     continue;
                 }
-
+                
```

Comment:
I assume that these style issues are external?

Reply:
Yes. Rollup leaves the spaces, but git or GitHub removes them again, I think. This seems to happen every time we update lint-md.js.

Reply:
I’m pretty sure it’s `git node land` that does that by passing `--whitespace=fix` to `git am`.